static/js/message_edit.js

```diff
@@ -157,6 +157,9 @@ function timer_text(seconds_left) {
 
 function edit_message(row, raw_content) {
     row.find(".message_reactions").hide();
+    if (row.find(".could-be-condensed").length !== 0) {
+        row.find(".message_expander").hide();
+    }
```

Comment:
I think it'd be structurally good for this to call an exported method in `condense.js`, so the logic around what the classes are called is all done via that module.  Then we're less likely to break this the next time we refactor.